update-dependencies/Config.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dotnet.Docker.Nightly
+{
+    public class Config
+    {
+        public static Config s_Instance { get; } = new Config();
+
+        private Lazy<string> _userName = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_USER"));
+        private Lazy<string> _email = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_EMAIL"));
+        private Lazy<string> _password = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_PASSWORD"));
+        
+        private Lazy<string> _cliVersionUrl = new Lazy<string>(() => GetEnvironmentVariable("CLI_VERSION_URL", "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/cli/rel/1.0.0"));
```

Comment:
>"https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/cli/rel/1.0.0" [](start = 111, length = 90)

Is this default useful?  I ask this because the value appears to only be applicable for v1 instead of the long term.  I would not recommend utilizing this default in our usage therefore I wonder if it is valuable.